easybuild/easyconfigs/b/BLAST/blast-2.2.30+.patch

```diff
@@ -0,0 +1,12 @@
+diff -ru ncbi-blast-2.2.30+-src.orig/c++/src/build-system/Makefile.in.top ncbi-blast-2.2.30+-src/c++/src/build-system/Makefile.in.top
```

Comment:
please add a brief description at the top describing what this patch fixes, and add yourself as an author